schema.sql

```diff
@@ -45,4 +45,33 @@ ADD COLUMN owner_id INT;
 SELECT * FROM animals;
 COMMIT;
 ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
-ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id);
+
+/* Create a link table named vets */
+CREATE TABLE vets (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name CHAR(25),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+/* Create a link table named specializations */
+CREATE TABLE specializations (
+  vet_id INT,
+  animal_id INT,
+  PRIMARY KEY (vet_id, animal_id),
+  FOREIGN KEY (vet_id) REFERENCES vets(id),
+  FOREIGN KEY (animal_id) REFERENCES animals(id)
+);
```

Comment:
🤔 Nice attempt on the specializations table! However, there's a slight modification needed to meet the project requirement.

- [ ] Currently, the specializations table represents a many-to-many relationship between animals and vets. According to the project requirement, it should represent a many-to-many relationship between species and vets. Please adjust the query accordingly to establish the correct relationship.

- [ ] It's crucial to ensure that your changes align with the project specifications. I recommend running the queries in both the `data.sql` and `queries.sql` files to verify if they produce the expected results. If necessary, update your code and include the updated screenshots in the PR Description.

Thank you for your dedication! Making these adjustments will bring your implementation in line with the project requirements. Keep up the good work! 👍🚀

Reply:
> 🤔 Nice attempt on the specializations table! However, there's a slight modification needed to meet the project requirement.
> 
> * [x]  Currently, the specializations table represents a many-to-many relationship between animals and vets. According to the project requirement, it should represent a many-to-many relationship between species and vets. Please adjust the query accordingly to establish the correct relationship.
> * [x]  It's crucial to ensure that your changes align with the project specifications. I recommend running the queries in both the `data.sql` and `queries.sql` files to verify if they produce the expected results. If necessary, update your code and include the updated screenshots in the PR Description.
> 
> Thank you for your dedication! Making these adjustments will bring your implementation in line with the project requirements. Keep up the good work! 👍🚀

According to your previous reviewer please kindly Currently, the specializations table represents a many-to-many relationship between animals and vets. According to the project requirement, it should represent a many-to-many relationship between species and vets. Please adjust the query accordingly to establish the correct relationship.